custom_auth/migrations/0004_create_manager.py

```diff
@@ -21,8 +21,12 @@ def make_permissions(apps, schema_editor):
 
     for perm in permissions:
         for action in ['add', 'view', 'change', 'delete']:
-            new_perm = Permission.objects.get(codename=f'{action}_{perm["model"]}', content_type__app_label=perm['app_label'])
-            manager_group.permissions.add(new_perm)
+            try:
```

Comment:
You might want to pull last changed from develop, as this migration has been commented there, because it can break migrations in some cases. Not a big deal, since it is not changing DB structure, but only populate fields

Reply:
If it worked for you so far, then you can trigger the merge :)